[ACTP] update for simplified PAR onboarding (#34315)
DataDog/documentation: content/en/actions/private_actions/use_private_actions.md

1. Confirm that you have sufficient permissions to create Kubernetes resources in your cluster.
1. Follow the instructions provided in the app to:
    1. Enroll the runner and generate the config.
    1. Add the **Private Action Runner** to your Helm repositories.
    1. Install the Helm chart.
1. Run `kubectl get pods -w` and verify that the status of the Private Action Runner pod becomes `Ready`.

{{% /tab %}}
{{< /tabs >}}

### Set up programmatically

As an alternative to the UI-based setup, you can enroll and configure a private action runner programmatically using your [API key][19] and [Application key][20]. This approach is ideal for automated deployments, CI/CD pipelines, and infrastructure-as-code workflows.

To use this authentication method:
1. Provide your Datadog API and App keys through the `DD_API_KEY` and `DD_APP_KEY` environment variables.
2. Pass the `--with-api-key` flag to the runner container.

The runner uses these credentials to register itself with your Datadog organization and assign the App key author as the runner editor.

#### Example commands

> Use the following commands to create an auto-enrollment script that can be re-run as needed for automated deployments. Once auto-enrollment succeeds, the runner appears on the **Private Action Runners** page.

Reviewer: ```suggestion
Use the following commands to create an auto-enrollment script that can be rerun for automated deployments. After the runner enrolls successfully, it appears on the **Private Action Runners** page.
```
